pkg/api/types.go

```diff
@@ -1025,6 +1025,13 @@ const (
 	// Status code 409
 	StatusReasonConflict StatusReason = "Conflict"
 
+	// StatusReasonResourceConflict means the requested update operation cannot be completed
+	// due to a resource conflict in the operation. The client may need to alter the request.
```

Comment:
How about "due to conflicting usage of underlying system/infrastructure resources".